working_time/working_time/doctype/working_time/working_time.js

```diff
@@ -43,3 +43,20 @@ frappe.ui.form.on("Working Time Log", {
 		frappe.model.set_value(cdt, cdn, "to_time", ""); // Otherwise Frappe may overwrite empty values with the current time on save.
 	},
 });
+
+// set billable time to 0% if Project is of Type "Internal", reset to 100% in case Project is "External"
+frappe.ui.form.on("Working Time Log", {
+	project: function(frm,cdt,cdn){
```

Comment:
```suggestion
frappe.ui.form.on("Working Time Log", {
	project: function (frm, cdt, cdn) {
		// set billable time to 0% if Project is of Type "Internal", reset to 100% otherwise
```

- Formatierung (Abstand nach Parametern)
- Kommentar näher am Code, den er beschreibt